client/tests/test_spark_provider.py

```diff
@@ -86,6 +86,32 @@ def transformation():
     )
 
 
+@pytest.mark.parametrize(
+    "sql",
+    [
+        ("SELECT * FROM {{test_name.test_variant}}"),
+    ]
+)
+def test_sql_transformation_without_variant(sql, spark_provider):
+    def transformation():
+        """doc string"""
+        return sql
+
+    decorator = spark_provider.sql_transformation()
+    decorator(transformation)
+
+    assert decorator.to_source() == Source(
+        name=transformation.__name__,
+        variant="default",
```

Comment:
we should also be able to not pass in a variant right?

Reply:
This does already default to `"default"`, so we _could_ leave it out.

Reply:
It does, just wanted to be explicit because it is a test case